src/components/graph/graph-types.ts

```diff
@@ -0,0 +1,29 @@
+export type uuid = string
+export type slug = string
+
+export type nodeData = {
```

Comment:
```suggestion
export interface NodeData {
```

Reply:
turns out I am not using this anymore. it's been deleted